x-pack/plugin/sql/sql-client/src/main/java/org/elasticsearch/xpack/sql/client/HttpClient.java

```diff
@@ -92,7 +96,7 @@ public boolean queryClose(String cursor) throws SQLException {
         byte[] requestBytes = toXContent(request);
         String query = "error_trace&mode=" +
                         request.mode() +
-                        (request.restClient() != null ? "&client=" + request.restClient().toString() : "");
+                        (request.clientId() != null ? "&clientid=" + request.clientId() : "");
```

Comment:
Use a constant for client id (by declaring it inside `ConnectionConfiguration`).
To have consistent naming, use `client.id` instead of `clientid`